include/sparrow/layout/primitive_array.hpp

```diff
@@ -295,26 +331,45 @@ namespace sparrow
     template <class T>
     template <std::ranges::input_range VALUE_RANGE, validity_bitmap_input R>
         requires(std::convertible_to<std::ranges::range_value_t<VALUE_RANGE>, T>)
-    arrow_proxy primitive_array<T>::create_proxy(VALUE_RANGE&& values, R&& validity_input)
+    arrow_proxy primitive_array<T>::create_proxy(
+        VALUE_RANGE&& values,
+        R&& validity_input,
+        std::optional<std::string_view>&& name,
+        std::optional<std::string_view>&& metadata
+    )
     {
         u8_buffer<T> data_buffer(std::forward<VALUE_RANGE>(values));
-        return create_proxy(std::move(data_buffer), std::forward<R>(validity_input));
+        return create_proxy(
+            std::move(data_buffer),
+            std::forward<R>(validity_input),
+            std::forward<std::optional<std::string_view>>(name),
+            std::forward<std::optional<std::string_view>>(metadata)
```

Comment:
no need for `std::forward` here, prefer `std::move`.

Reply:
done